core/page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/support/capabilities/SelenoidCapabilitiesParser.java

```diff
@@ -1,8 +1,11 @@
 package ru.sbtqa.tag.pagefactory.support.capabilities;
 
+import java.text.SimpleDateFormat;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.UUID;
+
```

Comment:
Do not insert a blank lines between imports, please

Reply:
Done